src/Task3Test.kt

```diff
@@ -0,0 +1,23 @@
+import org.junit.jupiter.api.Assertions.*
+import org.junit.jupiter.api.Test
+
+internal class Task3Test {
+
+    @Test
+    //Exceptional case where index -1 is returned
+    // to show that an array of length of less than
+    // 2 was given as input.
+    fun onEmptyArray() = assertTrue(!hasDuplicates(intArrayOf()))
+
+    @Test
+    // Same bad input scenario.
```

Comment:
More like: 'same bad comment', the input is fine you should be able to pass and array with 1 element and it will obviously have no duplicates :)